combiner/terraform/generate-combiner.tf

```diff
@@ -0,0 +1,73 @@
+# ECR
+resource "aws_ecr_repository" "combiner" {
```

Comment:
ECR usage is not strictly necessary. Since we are publishing the source code to github.com we can also publish the docker images to ghcr.io and pull the public images instead of needing to put them in an ECR in our AWS accounts.

Reply:
Created GitHub Issue to track work.